src/layouts/default.vue

```diff
@@ -9,6 +9,42 @@
     </q-header>
 
     <q-drawer v-model="leftDrawerOpen" show-if-above side="left" bordered>
+      <div class="column q-pl-lg justify-between window-height">
+        <div class="q-pt-xl">
+          <div class="column q-pb-xl items-start">
+            <p class="text-subtitle1 font-weight-bold">自分のルーム</p>
+            <q-btn flat align="left" class="q-pa-none q-py-sm"
+              >ディジェクマクン</q-btn
+            >
```

Comment:
これまで修正できてなさそうなのでやってもらえるとありがたい🙏
こんな感じで書くとうまくいきそう↓
```suggestion
            <q-btn flat align="left" class="q-pa-none q-py-sm full-width"
              >ディジェクマクン</q-btn
            >

```